fuzzers/build_google_oss_fuzzers.sh

```
@@ -66,13 +106,19 @@ package_runtime_libs()
     fi
 }
 
-fuzzerFiles=$(dirname $0)/*.cpp
+fuzzerFiles=$(dirname "$0")/*.cpp
```

Shellcheck (0.11.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)